roles/confluent.variables/vars/main.yml

```diff
@@ -22,6 +22,7 @@ zookeeper:
   systemd_file: /usr/lib/systemd/system/{{zookeeper_service_name}}.service
   systemd_override: /etc/systemd/system/{{zookeeper_service_name}}.service.d/override.conf
   log4j_file: "{{ archive_config_base_path if installation_method == 'archive' else '' }}/etc/kafka/zookeeper-log4j.properties"
+  log4j_root_logger: INFO, zkAppender
```

Comment:
can these be moved into `roles/confluent.<component>/defaults/main.yml` ? And let's keep the default as is, so stdout still in there. 

I personally like sdout, but I understand why this PR is created. Point being lets keep the default behavior the same, but allow for the customization!

Also don't put into a dictionary. we have variables put into dictionaries for no reason in our code

Reply:
Got it. Moved vars to role's default file.